src/main/java/rx/internal/schedulers/NewThreadWorker.java

```diff
@@ -30,24 +34,111 @@
     private final ScheduledExecutorService executor;
     private final RxJavaSchedulersHook schedulersHook;
     volatile boolean isUnsubscribed;
-
+    /** The purge frequency in milliseconds. */
+    private static final String FREQUENCY_KEY = "io.reactivex.rxjava.scheduler.jdk6.purge-frequency-millis";
```

Comment:
I'd like to fix it in master but then it conflicts with #2579, which by itself also adds parameters with unconventional naming.